### DISCUSSION TITLE
bindings

### DISCUSSION THREAD

**Discussion by unicomp21** [alessiodionisi/vmkit#2] [Ideas]
How were the go bindings generated?  I would love drive macos vm's using dart, wondering how hard it would be to do.  docker-desktop on mac can be frustrating at times.  my preference would be to run linux vm(s) on macos, and then run docker inside those vm's.  for linux devs using macs, i think this could simplify some things, assuming the performance is sufficient.

**Comment by alessiodionisi** [comment]
I think you can execute VMKit commands from Dart but the project is not yet stable. Currently it uses only QEMU since in the tests I did Apple virtualization framework has some stability problems that I hope will be solved with the next version of macOS.
